tools/tests/test_base.py

```diff
@@ -150,6 +150,7 @@ def __init__(self, command='../osqueryi', args={}, env={}):
         options["database_path"] += str(random.randint(1000, 9999))
         command = command + " " + " ".join(
             ["--%s=%s" % (k, v) for k, v in options.iteritems()])
+        print(command)
```

Comment:
Is this needed? I'd say remove this for production, or add some additional flair to make it clear it's for debugging purposes.